tmva/sofie/CMakeLists.txt

```diff
@@ -41,6 +41,7 @@ ROOT_STANDARD_LIBRARY_PACKAGE(ROOTTMVASofie
    TMVA/ROperator_Custom.hxx
    TMVA/ROperator_LayerNormalization.hxx
    TMVA/ROperator_Expand.hxx
+   TMVA/ROperator_Swish.hxx
```

Comment:
We should remove this from here?

Reply:
Okay, this is required for fixing issue mentioned here
https://github.com/root-project/root/pull/12945#issuecomment-1592685977